android/java/org/chromium/chrome/browser/homepage/settings/BraveRadioButtonGroupHomepagePreference.java

```diff
@@ -0,0 +1,109 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.homepage.settings;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.widget.RadioGroup;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.VisibleForTesting;
+import androidx.preference.PreferenceViewHolder;
+
+import org.chromium.chrome.R;
+import org.chromium.components.browser_ui.widget.RadioButtonWithDescription;
+import org.chromium.components.browser_ui.widget.RadioButtonWithEditText;
+
+/** Brave's version of a radio button group Preference used for Homepage Preference. */
+public final class BraveRadioButtonGroupHomepagePreference
+        extends RadioButtonGroupHomepagePreference {
+    static final String MOBILE_BOOKMARKS_PATH = "chrome-native://bookmarks/folder/1";
```

Comment:
Is `MOBILE_BOOKMARKS_PATH` package private and not private because of the test?

Reply:
Exactly

Reply:
Addressed here 067c9b454b850a75252f156f812d3bef4c54eb97